docs/intelligentapps/overview.md

```diff
@@ -1,34 +1,110 @@
+
+### AI Engineers & Data Scientists
+* **AI engineers** fine-tuning models for specific domains and deploying to production
+* **Data scientists** evaluating model performance and comparing different approaches
+* **ML engineers** converting and optimizing models for efficient local deployment
+
+### Researchers & Educators
+* **AI researchers** experimenting with different models and prompt engineering techniques
+* **Educators** teaching AI concepts and demonstrating model capabilities
+* **Students** learning about generative AI and hands-on model interaction
+
+### Key Use Cases
+* Explore and evaluate models from providers like Anthropic, OpenAI, and GitHub
+* Run models locally using ONNX and Ollama for privacy and cost control
+* Build and test agents with prompt generation and MCP tool integrations
+* Convert and optimize models for deployment across different hardware configurations
 
 ## Install and setup
 
-You can install the AI Toolkit from the Extensions view in VS Code:
+### Quick Installation
+
+The fastest way to get started is through the VS Code Extensions marketplace:
 
 > <a class="install-extension-btn" href="vscode:extension/ms-windows-ai-studio.windows-ai-studio">Install the AI Toolkit for VS Code</a>
 
-You can switch the installation between the formal released version for stable features and pre-released version for early access of new features. Check What's New during the installation for detailed feature list of each version.
+### Manual Installation
+
+1. Launch Visual Studio Code.
+2. Open the Extensions view by selecting **View** > **Extensions** or pressing **Ctrl+Shift+X**.
+
+   Alternatively, select the Extensions icon from the VS Code Activity Bar.
+
+3. Search for **AI Toolkit** in the search box.
+
+   ![install](./images/overview/install_1.png)
+
+4. Select **AI Toolkit for Visual Studio Code** from the search results and select **Install**.
+
+   ![install](./images/overview/install_2.png)
+
+5. After successful installation, the AI Toolkit icon appears in the VS Code Activity Bar.
+
+   ![install](./images/overview/install_4.png)
+
+> [!TIP]
+> Choose between the **Release** version for stable features or the **Pre-Release** version for early access to new capabilities. You can switch between versions anytime after installation.
+
+
+### Version Management
+
+#### Switch to a Specific Version
+
+By default, VS Code automatically updates AI Toolkit. To install a specific version:
+
+1. Open the **Extensions** view from the VS Code Activity Bar.
+2. Search for **AI Toolkit**.
+3. On the AI Toolkit page, select the dropdown arrow next to **Uninstall**.
+4. Choose **Install Specific Version...** from the dropdown.
+5. Select your desired version from the list.
+
+   ![install](./images/overview/install_5.png)
+
+> [!TIP]
+> Check the **What's New** section during installation to see detailed features for each version.
+
+#### Install Pre-Release Version
```

Comment:
Same as above, this info is also available in the Extension Marketplace doc: https://code.visualstudio.com/docs/configure/extensions/extension-marketplace#_install-a-prerelease-extension-version